src/components/FormErrors.js

```diff
@@ -0,0 +1,21 @@
+import React, { Component } from 'react';
+
+class FormErrors extends Component {
+
+  render(){
+
+      return(
+      <div className='formErrors'>
+        {Object.keys(this.props.formErrors).map((fieldName, i) => {
+          if(this.props.formErrors[fieldName].length > 0){
+            return (
+              <p key={i}>{this.props.formErrors[fieldName]}</p>
+            )
+          } else {
+            return '';
+          }
+        })}
+      </div>
+  )}
+}
+export default FormErrors;
```

Comment:
Do we need this component? If you specify the url input as type="url", the browser will automatically give the user an error message when submitting a bad url